zipper/types/response.go

```diff
@@ -117,74 +117,50 @@ func swapFetchResponses(m1, m2 *protov3.FetchResponse) {
 	m1.StopTime, m2.StopTime = m2.StopTime, m1.StopTime
 }
 
-func MergeFetchResponses(m1, m2 *protov3.FetchResponse, uuid string) *errors.Errors {
```

Comment:
The general cases that happens quite often are:
1. There is a different size because clock is different and we have one extra point in one of the metrics. It actually must be addressed as it's one of the most annoying one.
2. Second case where start point differs, it's quite rare though, but might happen with different storages. Also should be addressed.
3. Third case is where steps differs, in that case logic should be more clever, but see comment in next commit about that. It should be mixture of the one that's in Graphite now and some clever heuristics - we should mark as deffective metrics where there are a lot of NaNs. Maybe have a logic to determine real timestep for data and prefer it.